FusionIIIT/applications/academic_procedures/views.py

```diff
@@ -312,7 +312,8 @@ def academic_procedures_student(request):
         if user_sem==2 and des_flag==False and ug_flag==True:
             branchchange_flag=True
 
-        pre_registration_date_flag = get_pre_registration_eligibility(current_date)
+        pre_registration_date_flag = get_pre_registration_eligibility(current_date, user_sem, year)
+        # print("PRD: ", pre_registration_date_flag)
```

Comment:
```suggestion
        # print("PRD: ", pre_registration_date_flag)
```
Kindly erase all the comments or print statements added for the debugging in the PR.